### PR TITLE
Bug/UI/tra 4169 apply qury by enter

### PR DESCRIPTION
**Pull request by leon-up9**

### CODE REVIEW

**Review by mertyildiran — COMMENTED** [review]
It seems like the query is not submitted when I press <kbd>CTRL</kbd>+<kbd>ENTER</kbd>

FYI: I'm on Linux/Ubuntu 20.04 LTS, Google Chrome Version 97.0.4692.71 (Official Build) (64-bit)

**Comment by leon-up9** [comment]
@mertyildiran As far as I know the code wasn't published yet

File: ui/src/components/Filters.tsx [hunk 1]
```diff
@@ -60,7 +62,10 @@ export const QueryForm: React.FC<QueryFormProps> = ({backgroundColor, ws, openWe
         setQuery(e.target.value);
     }
 
+    
+
     const handleSubmit = (e) => {
+        debugger;
```
Comment (lirazyehezkel) [inline ui/src/components/Filters.tsx:68]: remove debugger

**Review by lirazyehezkel — COMMENTED** [review]
looks like a nice hook.
we have some more places in the code with enter key press, I would refactor those parts as well to use this hook. wdyt?

**Comment by leon-up9** [comment]
html element

**Comment by leon-up9** [comment]
Sure i will find them and refactor to use it?

File: ui/src/configs/shortcutsKeyboard.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const dictionary = {
+    "QuerySubmit" : [{metaKey : true, code:"Enter"}, {ctrlKey:true, code:"Enter"}]
```
Comment (lirazyehezkel) [inline ui/src/configs/shortcutsKeyboard.ts:2]: what are metaKey and ctrlKey?

Comment (leon-up9) [reply ui/src/configs/shortcutsKeyboard.ts:2]: the properties on the KeyboardEvent  and "rules" which I compare against

File: ui/src/hooks/useKeyPress.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { useCallback, useEffect, useLayoutEffect, useRef } from 'react';
+
+const useKeyPress = (eventConfigs, callback, node = null) => {
```
[the remark this answers is not in the document]
Comment (leon-up9) [reply ui/src/hooks/useKeyPress.ts:3]: html element

**Comment by lirazyehezkel** [comment]
> 

go for it

File: ui/src/configs/shortcutsKeyboard.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const dictionary = {
+    ctrlEnter : [{metaKey : true, code:"Enter"}, {ctrlKey:true, code:"Enter"}],
```
Comment (lirazyehezkel) [inline ui/src/configs/shortcutsKeyboard.ts:2]: add a comment with: support Ctrl/command

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**